### PR TITLE
Update to origami v2

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -1,9 +1,6 @@
 {
-	"description": "Provides lazy loading functionality for images, pictures, iframes, and more. It is powered by the Intersection Observer API to detect when the visibility of elements changes.",
-	"keywords": ["lazy", "loading", "images"],
-	"origamiType": "module",
-	"origamiCategory": "components",
```
Comment (JakeChampion) [inline origami.json:5]: There is a proposal to remove `origamiCategory` but we haven't accepted it just yet -- https://github.com/Financial-Times/origami/pull/120

File: .github/workflows/publish-to-npm-as-latest.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  publish-latest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2.1.5
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: npm ci
```
Comment (JakeChampion) [inline .github/workflows/publish-to-npm-as-latest.yml:15]: I only just noticed this now, after reviewing quite a few of these pull-request but, I don't think we need the `npm ci` step here, there's no harm having it either though

I think I added this step when I was investigating how to publish typescript `.d.ts` files for our components, which meant adding typescript as a dependency and installing it before running `npm publish

As we don't run any pre-publish scripts for our components, we don't need to run `npm ci` in this workflow or the prerelease workflow

File: origami.json [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
-	"origamiVersion": 1,
+	"origamiType": "component",
```
Comment (chee) [inline origami.json:2]:
```suggestion
	"origamiType": "module",
```

File: origami.json (hunk 1 above)
Comment (JakeChampion) [reply origami.json:5]: It's now been accepted 👍

File: package.json [hunk 4]
```diff
@@ -1,18 +1,23 @@
 {
-	"private": true,
 	"name": "o-lazy-load",
```
Comment (JakeChampion) [inline package.json:2]:
```suggestion
	"name": "@financial-times/o-lazy-load",
```

**Comment by JakeChampion** [comment]
@Financial-Times/origami-core - this is ready to review 👍

File: README.md [hunk 5]
```diff
@@ -152,8 +152,9 @@ o-lazy-load features may be included granularly using an options `$opts` map:
 
 State | Major Version | Last Minor Release | Migration guide |
 :---: | :---: | :---: | :---:
-✨ active | 2 | N/A | [migrate to v2](MIGRATION.md#migrating-from-v1-to-v2) |
-⚠ maintained | 1 | 1.0 | - |
+✨ active | 3 | N/A | [migrate to v3](MIGRATION.md#migrating-from-v2-to-v3) |
+⚠ maintained | 2 | 2.0.7 | [migrate to v2](MIGRATION.md#migrating-from-v1-to-v2) |
+╳ deprecated | 1 | 1.0 | - |
 
 ---
```
Comment (chee) [inline README.md:159]:
```suggestion
***
```

**Review by chee — COMMENTED** [review]
i approve

**Review by JakeChampion — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v3.0.0 :tada:

**Pull request merged**